src/OrchardCore/OrchardCore.DisplayManagement/Notify/NotifyFilter.cs

```diff
@@ -137,7 +138,12 @@ private void DeleteCookies(ResultExecutingContext filterContext)
 
         private IDataProtector CreateTenantProtector()
         {
-            return _dataProtectionProvider.CreateProtector(nameof(NotifyFilter), _tenantPath);
+            if (_dataProtectionProvider is ITenantDataProtectionProvider provider)
```

Comment:
@sebastienros FYI I opted for this approach as `NotifyFilter` is not in a module (so we can't be sure the DataProtection module was enabled).